lib/registries/addon/overview/styles.scss

```diff
@@ -55,3 +55,9 @@
     padding: 20px;
     min-width: 300px;
 }
+
+.NoticeText {
+    color: $primary-red;
+    font-weight: bold;
+    text-transform: uppercase;
```

Comment:
We generally avoid using all caps even when it's what the mockups have. With red and bold, imo, it's unnecessary to have the text in all caps. No worries if Product insists.